whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -71,7 +74,7 @@ public void setup() {
                 "upd-to:        noreply@ripe.net\n" +
                 "auth:          MD5-PW $1$d9fKeTr2$Si7YudNf4rUGmR71n/cqk/ #test\n" +
                 "mnt-by:        OWNER-MNT\n" +
-                "referral-by:   OWNER-MNT\n" +
+                "mbrs-by-ref:   OWNER-MNT\n" +
                 "created:         2011-07-28T00:35:42Z\n" +
```

Comment:
can you also add test with multiple value of same attribute? This should have multiple redaction with different handle for same attrbute , if I am not mistaken

Reply:
Done